app/views/content/is-teaching-right-for-me/maths/maths/_article.html.erb

```diff
@@ -0,0 +1,95 @@
+    </p>
+
+    <p>
+      An exciting career spent passing your knowledge on to the next generation could be for you.
+    </p>
+
+  </section>
+
+  <section class="col col-space-s col-content-max">
+    <%= render TextBlockWithTwoImagesComponent.new(
+      heading: "quote",
+      colour: "pink",
+      image_1_path: "static/images/content/hero-images/0017.jpg",
+      image_1_alt: "",
+      image_2_path: "static/images/content/hero-images/0024.jpg",
+      image_2_alt: "",
+    ) do %>
+      <p>
+        quote      </p>
+    <% end %>
+  </section>
+
+  <section class="col col-720 col-horizontal col-space-m">
+    <div>
+      <h2>What you'll be teaching</h2>
+      <p>You'll teach the national curriculum. The themes you're likely to cover include: </p>
+      <p>Themes you’ll cover for 11 to 16 year olds (key stage 3 and 4):</p>
+      <ul class="vertical-tags">
+        <li class="tag">Numbers</li>
+        <li class="tag">Algebra</li>
+        <li class="tag">Ratio, proportion and rates of change</li>
+        <li class="tag">Geometry and measures</li>
+        <li class="tag">Probability and statistics</li>
+      </ul>
+
+      <p>Themes you’ll cover for 16 to 18 year olds (key stage 5) - A or AS level core maths:</p>
+      <ul class="vertical-tags">
+        <li class="tag">Analysis of data</li>
+        <li class="tag">Maths for personal finance</li>
+        <li class="tag">Estimation</li>
+        <li class="tag">Critical analysis of given data and models</li>
+      </ul>
+    <p><a href="https://www.gov.uk/government/publications/national-curriculum-in-england-mathematics-programmes-of-study/national-curriculum-in-england-mathematics-programmes-of-study#key-stage-3">Read the full national curriculum for more information about teaching maths in secondary schools</a>.</p>
+    <p>You may also get the opportunity to teach other subjects, particularly for 11 to 16 year olds and in some cases for post-16. This will vary depending on individual schools.
+    </div>
+</section>
+<section class="col col-845 inset">
+    <%= render Content::PhotoQuoteListComponent.new(
+      [
+        {
+          image: "static/images/content/subjects/rotna-roy.jpeg",
+          heading: "Empowering my pupils",
+          text: "I enjoy showing my pupils that maths is essential in everyday life. Whether it is identifying patterns, interpreting data, managing finances, or understanding the spaces and structures around us, everything uses maths.",
+          accreditation: "Rotna Roy, Maths Lead at a school in London",
+        }
+      ],
+      colors: %w[yellow],
+      numbered: false
+    ) %>
+  </section>
+
+<section class="col col-space-m col-720">
+<h2>Fund your teacher training</h2>
+<p>To train to be a maths teacher, you will need to do a teacher training course. </p>
```

Comment:
Do we need this sentence? It doesn't really make sense?

Reply:
I personally think this should be here, in case users have come here before looking at funding

Reply:
@caitlin-reidy I think perhaps we could maybe reword it to: If you want to be a maths teacher you'll need to do a teacher training course

Or something along those lines. I think it's the first part of the sentence that's maybe confusing?